options-pattern/src/DiyConfigExample/StaticConfigRoot.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using System;
+
+namespace DiyConfigExample
+{
+    static class StaticConfigRoot
+    {
+        private static IConfigurationRoot _instance;
+
+        public static IConfigurationRoot Instance
+        {
+            get
+            {
+                if (_instance == null)
+                {
+                    _instance = new ConfigurationBuilder()
+                        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
this doesn't build out of the box for me- nuget restore has no issues getting the packages

the proj file has no mention of Microsoft.Extensions.Configuration.Json either.

Reply:
Thanks, resolved.